Marlin/language_de.h

```diff
@@ -153,6 +159,12 @@
 #define MSG_ERR_MINTEMP                     LCD_STR_THERMOMETER " UNTERSCHRITTEN"
 #define MSG_ERR_MAXTEMP_BED                 "BETT " LCD_STR_THERMOMETER " ÜBERSCHRITTEN"
 #define MSG_ERR_MINTEMP_BED                 "BETT " LCD_STR_THERMOMETER " UNTERSCHRITTEN"
+#define MSG_ERR_Z_HOMING                    "G28 Z Forbidden"
```

Comment:
The concept is:
* All strings in language_en.h can be translated.
* No string in language.h shall be translated.